shipment_management/public/js/custom_script.js

```diff
@@ -24,20 +24,21 @@ frappe.ui.form.on("Delivery Note", {
 });
 
 function create_dialog(frm) {
-    args = []
-    box_string = ""
-    var count = 1;
-    for (var i = 0; i < frm.doc.items.length; i++) {
-        for (var j = 0; j < frm.doc.items[i].qty; j++) {
+    let args = [];
+    let box_string = "";
+    let count = 1;
+
+    for (let i = 0; i < frm.doc.items.length; i++) {
+        for (let j = 0; j < frm.doc.items[i].qty; j++) {
             box_string += "Box " + count + "\n";
             count++;
         }
     }
 
-    var item_count = 1;
-    item_dict = {}
-    for (var i = 0; i < frm.doc.items.length; i++) {
-        for (var j = 0; j < frm.doc.items[i].qty; j++) {
+    let item_count = 1;
+    let item_dict = {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'item_dict' is not in camel case. (camelcase)](https://app.codacy.com/manual/mail_67/shipment_management/pullRequest?prid=5453114)